app/src/main/java/org/fdroid/fdroid/Preferences.java

```diff
@@ -234,8 +234,7 @@ public boolean isLocalRepoHttpsEnabled() {
     }
 
     private String getDefaultLocalRepoName() {
-        return (Build.BRAND + " " + Build.MODEL + new Random().nextInt(9999))
-                .replaceAll(" ", "-");
+        return "RAD F-DROID Repo";
```

Comment:
@patience-mpofu I'm unsure about this change. Firstly, we shouldn't be changing their Java code - we can override Java code in the same way as all the other files (by making a copy into the jembi directory), and secondly, it would be better to try use Build.BRAND, Build.MODEL to get a reasonable name to avoid us having to override the Java.

Reply:
Reverted to original https://github.com/jembi/fdroidclient/commit/16651cc416043e237e476f1b2348ed8c6733c95f